Dockerfile.worker

```diff
@@ -0,0 +1,18 @@
+FROM python:3.5-slim
+
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
+WORKDIR /home/user
+
+ENV CELERY_VERSION 4.0.2
+
+COPY workers/requirements.in .
+
+RUN pip install pip-tools && \
+    pip-compile requirements.in && \
+    pip install -r requirements.txt
+
+COPY workers/ .
+
+USER user
+
+CMD ["celery", "-A", "bioinformatics_mill_workers", "-c", "1", "worker"]
```

Comment:
It's not so much a pattern as a workaround I came up with to solve the issue I was having where I didn't yet have a foreman to actually send the workers messages, so I made a very simple foreman app (that's what `master.py` is) that just runs `save_test_batch.delay()` to make sure the containers are talking to each other.